app/frontend/package.json

```diff
@@ -17,6 +16,7 @@
     "react-dom": "^17.0.0",
     "react-hook-form": "^6.15.1",
     "react-intersection-observer": "^8.31.0",
+    "react-lines-ellipsis": "^0.14.1",
```

Comment:
Don't really like this dependency, though small, it has no types. But I didn't want to waste time solving this myself. (This being having ellipsis overflow with an arbitrary number of lines.)

Reply:
Saw that you typed it yourself - not popular enough for it to have types available on DefinitelyTyped?

Reply:
Sadly not